Tests/Rules/UseConsistentIndentation.tests.ps1

```diff
@@ -395,6 +395,12 @@ foo |
             @{ IdempotentScriptDefinition = @'
 foo |
     bar -Parameter1
+'@
+            },
+            @{ IdempotentScriptDefinition = @'
+Get-TransportRule |
+Select-Object @{ E = $SenderDomainIs | Sort-Object }
```

Comment:
should we include a test for multiple lines ?
```powershell
$files = get-ChildItem |
Where-Object {$_.name -match "a"}|
Where-Object {$_.length -gt 1mb } |
Foreach-Object { $_.FullName}
```

Reply:
yes, why not to make it more realistic, done now